src/sentry/tasks/backfill_seer_grouping_records.py

```diff
@@ -174,6 +202,10 @@ def backfill_seer_grouping_records(
             project, rows, group_id_message_batch_filtered, group_hashes_dict
         )
 
+        # If nodestore is down, we should stop
+        if data["data"] == [] and data["stacktrace_list"] == []:
+            return
```

Comment:
We already try multiple fallbacks and sleeps in between exceptions for getting data from nodestore - if we reach this point, we should stop trying (the exceptions are logged in the functions where they happen)